week03/css/about.css

```
@@ -0,0 +1,127 @@
+@font-face {
+    font-family: '국립박물관문화재단클래식B';
+    src: url('https://cdn.jsdelivr.net/gh/projectnoonnu/noonfonts_twelve@1.0/국립박물관문화재단클래식B.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: '국립박물관문화재단클래식M';
+    src: url('https://cdn.jsdelivr.net/gh/projectnoonnu/noonfonts_twelve@1.0/국립박물관문화재단클래식M.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+#header {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    padding: 10px 15px;
+    background-color: white;
+    color: #6a75ca;
+    font-size: 18px;
+    border-bottom: 3px solid #B4C7E7;
+}
+
+body {
+    background-color:#E6EEFB;
+}
+
+.title {
+    font-family: '국립박물관문화재단클래식B';
+    font-size: 40px;
+    color: black;
+    justify-content: center;
+    text-align: center;
+    display: flex;
+    margin: 250px 0 0 0;
+}
+
+.navbar {
+    background-color: white;
+    height: 75px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+
+}
+
+.nav1 {
+    float: left;
+    width: 20%;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+
+.logo img {
+    width: 170px;
+    height: 80px;
+}
+
+.nav2 {
+    float: left;
+    width: 50%;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+
+.nav3 {
+    float: left;
+    width: 40%;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+}
+
+.navbar a {
+    text-decoration: none;
+    color: gray;
+    font-weight: lighter;
+    font-size: 23px;
+}
+.navbar a:hover {
+    color: #0D51FE;
+}
+
+.navbar ul {
+    float: right;
+}
+
+.navbar li {
+    margin-left: 15px;
+    list-style-type: none;
+    float: left;
+    font-family: '국립박물관문화재단클래식B';
+}
+
+.navbar li:hover {
+    border-bottom: 2px solid #0D51FE;
+}
+
+.BLog {
+    display: inline-block;
+    margin: 100 0 0 350px;
+}

```

개개 요소에 마진을 주어 중앙 정렬을 맞추는 것보다는, 전체 요소를 포함하는 body 혹은 main 태그에 너비를 주고 양옆 margin을 auto로 설정함으로써 중앙 정렬을 할 수 있습니다.
또, `justify-content: space-evenly;` 속성을 활용하면 자식 요소 바깥과 사이의 간격을 동일하게 설정할 수 있습니다.

```suggestion
body main {
    width: 1200px;
    margin: 0 auto;
}
body main .content {
    display: flex;
    justify-content: space-evenly;
}
```